fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/query/FDBOrQueryToUnionTest.java

```diff
@@ -1970,13 +1972,13 @@ void testOrQuerySplitContinuations(OrQueryParams orQueryParams) throws Exception
                 .build();
         orQueryParams.setPlannerConfiguration(this);
 
-        // Index(MySimpleRecord$num_value_3_indexed [[1],[1]]) ∪ Index(MySimpleRecord$num_value_3_indexed [[3],[3]]) ∪ Index(MySimpleRecord$num_value_3_indexed [[5],[5]])
+        // Fetch(Covering(Index(MySimpleRecord$num_value_3_indexed [[1],[1]]) -> [num_value_3_indexed: KEY[0], rec_no: KEY[1]]) ∪[_.num_value_3_indexed, _.rec_no] Covering(Index(MySimpleRecord$num_value_3_indexed [[3],[3]]) -> [num_value_3_indexed: KEY[0], rec_no: KEY[1]]) ∪[_.num_value_3_indexed, _.rec_no] Covering(Index(MySimpleRecord$num_value_3_indexed [[5],[5]]) -> [num_value_3_indexed: KEY[0], rec_no: KEY[1]]))
         RecordQueryPlan plan = planQuery(query);
```

Comment:
this is the result of `System.out.println(plan)`, and it looks like this before my change. The real difference introduced by my change is that before the change, it is `concat(primaryKey, field("num_value_3_indexed")` and now the order is changed to `concat(field("num_value_3_indexed", primaryKey)`

Reply:
Why is there still a difference? I think this needs to be resolved before this can go in.

Reply:
Nevermind, this is just a comment. Please undo it as it does not reflect the current plan.